src/linter.js

```diff
@@ -634,21 +635,15 @@ export default class Linter {
   }
 
   _markBadwordUsage(filename, fileData) {
-    if (fileData) {
-      const words = fileData.split(' ');
-
-      for (let word of words) {
-        // Filter out non a-Z characters to simplify matching with our
-        // badword list.
-        word = word.toLowerCase().replace(/[^a-zA-Z]/, '').trim();
-
-        if (word && word.match(constants.BADWORDS_RE.en)) {
-          this.collector.addNotice(
-            Object.assign({}, messages.MOZILLA_COND_OF_USE, {
-              file: filename,
-            })
-          );
-        }
+    if (fileData && fileData.trim()) {
+      const sanitizedFileData = fileData.replace(/[^a-z]/g, '').trim();
```

Comment:
I think the `.trim()` is unnecessary since the `replace` should already have taken care of whitespaces?